package.json

```diff
@@ -28,9 +28,9 @@
     "test": "jest --coverage"
   },
   "dependencies": {
-    "@fortawesome/fontawesome-svg-core": "^1.2.21",
-    "@fortawesome/free-solid-svg-icons": "^5.10.1",
-    "@fortawesome/react-fontawesome": "^0.1.4",
+    "@fortawesome/fontawesome-svg-core": "^1.2.30",
```

Comment:
Any specific reason to upgrade font awesome?

Reply:
No, I think it happened in one of my many many many `npm install`s, and I saw no reason why not?